knative/deployer.go

```diff
@@ -388,6 +385,32 @@ func generateNewService(f fn.Function, decorator DeployDecorator) (*v1.Service,
 	return service, nil
 }
 
+// newServiceAnnotations creates a final map of service annotations based
+// on static defaults plus the function's defined annotations plus the
+// application of any provided annotation decorator.
+func newServiceAnnotations(f fn.Function, d DeployDecorator) (aa map[string]string) {
+	aa = make(map[string]string)
+
+	// Static default: Dapr support
+	aa["dapr.io/enabled"] = "true"
```

Comment:
should these be constants, eventually?

I'd perhaps also populate all dapr related annotations in a separated call :man_shrugging:

Reply:
+1, the same way we have constants for labels: https://github.com/knative/func/tree/main/k8s/labels

Reply:
also do we want to apply Dapr annotations to all functions by default?

Reply:
> should these be constants, eventually?

I see no reason to make constants unless the values are used in more than one place.

Reply:
Thanks for the revies, please see the latest for:  
- labels defined in `labels.go`
- Annotations are now extracted and added in the same manner as the functions' annotation map below (looping over the map keys) for consistency
- the annotations define how Dapr can connect its control plane to the service if it is installed, otherwise has no effect